client/starwhale/core/job/model.py

```diff
@@ -169,6 +170,9 @@ def execute(self) -> TaskResult:
                 # The standard implementation does not return results
                 func()
         except Exception as e:
+            logger.exception(
```

Comment:
`logger.exception(e)` will show the exception traceback.

Reply:
done